src/main/java/data/Vendor.java

```diff
@@ -64,4 +64,27 @@ public void addAccount(Account account) {
   public ArrayList<Account> getAccounts() {
     return accountList;
   }
+
+  /** Return a String representing a billing config's JSON content. */
+  public String createConfig() {
+    ArrayList<Account> accounts = getAccounts();
+    String config = toString();
+
+    for(int i = 0; i < accounts.size(); i++) {
+      config += accounts.get(i).toString();
+    }
+    config += "]}";
+
+    return config;
+  }
+
+  /** Return a String representation of Vendor fields. */
+  public String toString() {
```

Comment:
Again, I wouldn't overwrite toString if it's meant to be JSON.

Also use String.format and make sure that you're generating valid JSON.

Perhaps you should be using a JSON object throughout instead of a String? That would probably make things much nicer.

Reply:
I originally was going to use JSONObjects but was told that StringBuilder would be better. I'll switch it to String.format.